site/docs/v1/tech/apis/entity-management/entity-types.adoc

```diff
@@ -198,9 +198,9 @@ The database column to order the search results on plus the order direction.
 +
 The columns you can use for this are:
 +
-* `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Entity Type was created
-* `lastUpdateInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Entity Type was last updated
-* `name` - the name of the Entity Type
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Entity Type was created
```

Comment:
Would prefer in general if this type of fix (which is great, btw) happened in a separate PR, because it doesn't need much review (and, in a few months, when you're familiar with the system, no review).

Can you cherry pick this commit and submit as a sep PR?